packages/overlay/src/ActiveOverlay.ts

```diff
@@ -223,7 +223,10 @@ export class ActiveOverlay extends SpectrumElement {
     }
 
     public feature(): void {
-        this.tabIndex = -1;
+        // eslint-disable-next-line spectrum-web-components/document-active-element
+        if (!this.contains(document.activeElement)) {
```

Comment:
Allows you to tab through https://overlay-escape--spectrum-web-components.netlify.app/storybook/?path=/story/dialog-wrapped--tooltips in both forward and backward directions without getting blocked out of the Dialog after leaving the iframe...